### PR TITLE
upgrade to node 16

### PR DESCRIPTION
**Pull request by fabriciojs**
| Issue | - |
| -----: | :-----: |
| :warning: Break Change | No |

**Description**

Updating all templates to ship with Node 16 image references instead of old Node 14.

### CODE REVIEW

File: templates/app/node14-adonis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 services:
   app:
-    image: kooldev/node:14-adonis
+    image: kooldev/node:16-adonis
```
Comment (danielsuguimoto) [inline templates/app/node14-adonis.yml:3]: we should rename this file to `node-adonis.yml` and change its references as well

File: templates/app/node14-nestjs.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 services:
   app:
-    image: kooldev/node:14-nest
+    image: kooldev/node:16-nest
```
Comment (danielsuguimoto) [inline templates/app/node14-nestjs.yml:3]: we should rename this file to `node-nestjs.yml` and change its references as well

**Review by danielsuguimoto — CHANGES_REQUESTED** [review]
Nice! Minor comments about some files which name has the version 14 on it, but you changed the version to node 16. I think we should rename and make them more generic.

**Comment by fabriciojs** [comment]
@danielsuguimoto good catch! Have updated the PR accordingly.

**Review by danielsuguimoto — APPROVED** [review]

**Pull request merged**